css/responsive.css

```
@@ -58,4 +58,16 @@
       max-width: 311px;
       display: block;
    }
+
+   #introduce {
+      padding: 64px auto 0;
+   }
+
+   .section-title {
+      font-size: 21px;
+  }
+
+  .company-item img {
+      max-width: 158px;
+  }
```

```suggestion
  .company-item {
      max-width: 158px;
  }
```
nits
common.cssで`<img>`にwidth: 100%がついているので、ここは親への指定で大丈夫です！

---

ありがとうございます！
修正しました。

706ed53